datapackage_pipelines/web/server.py

```diff
@@ -220,27 +240,76 @@ def pipeline_api(field, pipeline_id):
     return jsonify(ret)
 
 
+def _make_badge_response(subject, text, colour):
```

Comment:
Just wondering why not do a redirect here.

Reply:
If it redirects away from the site, you can't change the url as easily, to see different results.